### ISSUE TITLE
Can't use navBarCustomView on Android?

### ISSUE THREAD

**Issue by koansang** [wix/react-native-navigation#2041]
### Issue Description

I'm use navBarCustomView. but no display on android.

### Steps to Reproduce / Code Snippets / Screenshots
My code: 
```js
this.props.navigator.setStyle({
      //navBarBackgroundColor: Colors.basic.blue,
      navBarCustomView: 'CustomNavBar',
      navBarComponentAlignment: 'center',
      //navBarCustomViewInitialProps: {title: 'Hi Custom'}
    });
```

### Environment
* React Native Navigation version: 1.1.236
* React Native version: 0.49.3
* Platform(s) (iOS, Android, or both?): android
* Device info (Simulator/Device? OS version? Debug/Release?): LG V20

**Comment by koansang** [comment]
I'm soleved.
set `navBarHeight`.

**Comment by guyca** [comment]
This should have been fixed in v1.1.243. Try updating to latest version, `navBarHeight` isn't needed.
